README.md

```diff
@@ -6,7 +6,64 @@
 [![Stackage Nightly](http://stackage.org/package/postgresql-simple-named/badge/nightly)](http://stackage.org/nightly/package/postgresql-simple-named)
 [![Build status](https://img.shields.io/travis/Holmusk/postgresql-simple-named.svg?logo=travis)](https://travis-ci.org/Holmusk/postgresql-simple-named)
 
-Implementation of named parameters for `postgresql-simple` library
+This library introduces the implementation of named parameters for the
+[`postgresql-simple`][pgs] library. `postgresql-simple-named` is considered to
+be used along with the [`postgresql-simple`][pgs] library, so you could refer
+there for the original documentation of primary functions. This package solves
+exclusively one particular problem — gives the ability to use named parameters
+instead of `?` in quasi-quoter queries and offers essential functions for using
```

Comment:
Jus to use `use` not so often 🙂 
```suggestion
instead of `?` in quasi-quoter queries and offers essential functions for substituting variables
```
(and delete `them` on next line)